pkgs/applications/misc/conduktor/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchurl, jdk11, unzip, makeWrapper, makeDesktopItem, copyDesktopItems }:
+
+stdenv.mkDerivation rec {
+  pname = "conduktor";
+  version = "2.7.2";
+
+  src = fetchurl {
+    url = "https://github.com/conduktor/builds/releases/download/v${version}/Conduktor-linux-${version}.zip";
+    sha256 = "1njyy27il9br2rd0i2q45yyyfvnsmsd6qm6j61522dmrki783kw4";
+  };
+
+  nativeBuildInputs = [ unzip makeWrapper copyDesktopItems ];
+
+  icon = fetchurl {
+    url = "https://github.com/conduktor/builds/raw/v${version}/.github/resources/Conduktor.png";
+    sha256 = "0s7p74qclvac8xj2m22gfxx5m2c7cf0nqpk5sb049p2wvryhn2j4";
+  };
+
+  desktopItems = makeDesktopItem {
+    type = "Application";
+    name = pname;
+    desktopName = "Conduktor";
+    genericName = meta.description;
+    exec = pname;
+    icon = icon;
+    comment = "A beautiful and fully-featured desktop client for Apache Kafka";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/applications
+    mv * $out
+    wrapProgram "$out/bin/conduktor" --set JAVA_HOME "${jdk11.home}"
+
+    runHook postInstall
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Apache Kafka Desktop Client";
+    longDescription = ''
+      Conduktor is a GUI over the Kafka ecosystem, to make the development
+      and management of Apache Kafka clusters as easy as possible.
+    '';
+    homepage = https://www.conduktor.io/;
+    changelog = "https://www.conduktor.io/changelog/#${version}";
+    license = stdenv.lib.licenses.unfree;
+    maintainers = with maintainers; [ trobert ];
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
Could you make `src` conditional on the platform? Does the linux zip provide support for multiple cpu types?

Reply:
the zip is not compatible with darwin (it's a specific .pkg binary) so I'll drop darwin for now, as I don't have a mac to test.